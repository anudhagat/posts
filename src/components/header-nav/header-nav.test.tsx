import React from 'react';
import { shallow } from 'enzyme';
import HeaderNav from './index';

describe('<HeaderNav/>', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<HeaderNav />);

    expect(wrapper.debug()).toMatchSnapshot();
  });
});
