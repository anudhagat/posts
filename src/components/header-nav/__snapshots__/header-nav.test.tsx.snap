// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderNav/> renders without crashing 1`] = `
"<header className=\\"header\\">
  <nav className=\\"nav\\">
    <ul className=\\"navList\\">
      <li className=\\"navListItem\\">
        <NavLink exact={true} to=\\"/\\" activeClassName=\\"selected\\">
          Posts
        </NavLink>
      </li>
      <li className=\\"navListItem\\">
        <NavLink to=\\"/users\\" activeClassName=\\"selected\\">
          Users
        </NavLink>
      </li>
    </ul>
  </nav>
</header>"
`;
